docs/_docs/plugins/installation.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Plugins
+permalink: /docs/plugins/installation/
+---
+
+You have 3 options for installing plugins:
+
+1. In your site source root, make a `_plugins` directory. Place your plugins
+   here. Any file ending in `*.rb` inside this directory will be loaded before
+   Jekyll generates your site.
+
+2. In your `_config.yml` file, add a new array with the key `plugins` (or `gems` for Jekyll < `3.5.0`) and the
+   values of the gem names of the plugins you'd like to use. An example:
+
+   ```yaml
+   # This will require each of these plugins automatically.
+   plugins:
+     - jekyll-gist
+     - jekyll-coffeescript
+     - jekyll-assets
+     - another-jekyll-plugin
+   ```
+
+   Then install your plugins using `gem install jekyll-gist jekyll-coffeescript jekyll-assets another-jekyll-plugin`
+
+3. Add the relevant plugins to a Bundler group in your `Gemfile`. An
```

Comment:
doesn't this still require adding to the `plugins` key?

Reply:
It doesn't for me. Having it in the bundler group automatically requires it.